### ISSUE TITLE
bug: Default answer based on previous questions does not work properly

### ISSUE THREAD

**Issue by Mat-moran** [copier-org/copier#513]
**Describe the problem**
The default value of a question based on previous question does not render properly.

**copier.yaml**
```
### Questions for the user
project_name:
  type: str

project_client:
  type: str

project_id:
  type: str
  default: "{{ project_name }}-{{ project_client }}"
```

**Template**

**To Reproduce**
1. Run `copier copy template project`
2. Answer to 1 -> project
3. Answet to 2 -> pepito
4. Default answer of 3 question --> None-None

**Expected behavior**
Default answer to 3 question should be `project-pepito`

**Screenshots/screencasts/logs**

**Environment**
 - OS: Pop-OS 20.04
 - Copier version: 6.0.0a6

**Comment by Yajo** [comment]
Can you confirm it happens in latest git master?

**Comment by Mat-moran** [comment]
@Yajo I can confirm that the issue is already fixed in version  v6.0.0a9!
Thanks men for maintaining this geat repo!!
